### DISCUSSION TITLE
Only Base Model on Whisper

### DISCUSSION THREAD

**Discussion by Pericu** [intel/openvino-plugins-ai-audacity#238] [General]
Hi guys,

Right now there only seems to be the "Base" model integrated. Weren't the other models part of it all as well in earlier versions? 
Can we implement other versions?

best,

**Comment by RyanMetcalfeInt8** [comment]
Hi @Pericu,

The latest version of the plugins has support for base, small, medium, and large variants of whisper model. During installation, you should be able to choose which models to download & install. 

Thanks!

**Reply by Pericu** [reply]
Ah, absolutely my bad! Completely missed it during the install. Thanks Ryan!
